operator/apis/machinelearning.seldon.io/v1/seldondeployment_webhook.go

```diff
@@ -39,6 +39,7 @@ var (
 	envPredictiveUnitGrpcServicePort    = os.Getenv(ENV_PREDICTIVE_UNIT_GRPC_SERVICE_PORT)
 	envPredictiveUnitServicePortMetrics = os.Getenv(ENV_PREDICTIVE_UNIT_SERVICE_PORT_METRICS)
 	envPredictiveUnitMetricsPortName    = GetEnv(ENV_PREDICTIVE_UNIT_METRICS_PORT_NAME, constants.DefaultMetricsPortName)
+	envAmbassadorEnabled                = GetEnv("AMBASSADOR_ENABLED", "false")
```

Comment:
Amend to variable to avoid magic word, same for default value (perhaps add on constants)